src/PgSqlHandle.php

```diff
@@ -241,13 +241,14 @@ private function send(callable $function, ...$args): \Generator
 
     /**
      * @param resource $result PostgreSQL result resource.
+     * @param string $sql SQL query text or statement id for error message
      *
      * @return \Amp\Sql\CommandResult|ResultSet
      *
      * @throws FailureException
      * @throws QueryError
      */
-    private function createResult($result)
+    private function createResult($result, &$sql)
```

Comment:
What's the reason for making this a reference?

Reply:
I've just want to save memory by preventing large SQL string copy.